app/src/main/java/ar/com/wolox/lucasdelatorre/training/activities/Signup.java

```diff
@@ -22,17 +28,24 @@
     private String mUsername;
     private String mPassword;
     private String mCPassword;
+    private User mUser;
+    private ActionBarActivity mActivity;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_signup);
 
+        init();
         setUi();
         populate();
         setListeners();
     }
 
+    private void init() {
+        mActivity = this;
```

Comment:
Para que definis esta variable?, por que no directamente donde la usas pones this?